app/assets/stylesheets/wedding_party.scss

```diff
@@ -0,0 +1,74 @@
+@import 'utils';
+@import 'variables';
+@import 'colors';
+@import 'font_faces/avenir';
+@import 'font-awesome';
+@import url(http://fonts.googleapis.com/css?family=Source+Code+Pro:400,200,300,500);
+
+.wedding-party:after, .wedding-party:before {
+  background-color: $light_grey;
+}
+
+.wedding-party {
+  span {
+    font-weight: 300;
+    font-size: 24px;
+    line-height: 100px;
```

Comment:
Looks like this value is used in a bunch of places--is that intentional? If so, would just store this in a variable so easy to change later / all at once if necessary.